app/assets/images/payment_icons/cimbthai.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-cimbthai">
+    <title id="pi-cimbthai">CIMB Thai</title>
+	<style>.st2{fill:#7d0b10}</style>
```

Comment:
Can you remove this line and add inline style to the target `path` directly? This would avoid unexpected overwritten styles. In the future, we will prevent adding `<style>`.